lib/pubsub/index.js

```diff
@@ -199,16 +199,10 @@ PubSub.prototype.getTopics = function(query, callback) {
  *
  * @example
  * pubsub.createTopic('my-new-topic', function(err, topic, apiResponse) {
- *   topic.publish({
- *     data: 'New message!'
- *   }, function(err) {});
```

Comment:
Out of curiosity, why are we removing this example? I understand the verbiage below, but this snippet still seems relevant.